config.json

```
@@ -671,8 +671,12 @@
         "slug": "gigasecond",
         "name": "Gigasecond",
         "uuid": "b7116dfb-1107-4546-9f95-f15acdb6f6a4",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "datetimes"
+        ],
+        "prerequisites": [
+          "numbers"
```

Currently, the `practices` and `prerequisites` values don't overlap at all. This is usually not what we want. Take this `gigasecond` exercise. `datetimes` is listed in the `practices` field, which is 100% correct. However, as Practice Exercises are not intended to teach Concepts but to help them practice a Concept that they've already learnt. Therefore, students should already be familiar with the `datetimes` Concept before being able to work on `gigasecond`. Thus, the `datetimes` Concept must also be a prerequisite:

```suggestion
          "datetimes",
          "numbers"
```

This comment also applies to the other exercises.